appeals/web/src/server/appeals/question-page/question-page.controller.js

```diff
@@ -0,0 +1,69 @@
+import { getAppealDetailsFromId } from '#appeals/appeal-details/appeal-details.service.js';
+import { getLpaQuestionnaireFromId } from '#appeals/appeal-details/lpa-questionaire/lpa-questionnaire.service.js';
+import { appealShortReference } from '#lib/appeals-formatter.js';
+import logger from '#lib/logger.js';
+import {
+	appealBackLink,
+	appealQuestionPage,
+	lpaQBackLink,
+	lpaQQuestionPage
+} from './question-page.mapper.js';
+
+/**
+ * @param {*} request
+ * @param {*} response
+ */
+const renderQuestionPage = async (request, response) => {
+	try {
+		const currentUrl = request.originalUrl;
+		let mappedData;
+		let data;
+		const appealId = request.params.appealId;
+		let backLink;
+		if (currentUrl.includes(`lpa-questionnaire/${request.params.lpaQId}/change`)) {
```

Comment:
I suppose we'll need to refactor this when implementing the appellant case. I suggest the idea of using a 'stage' field to identify the individual questionnaires.

Reply:
Yeah will have to when implementing APpellant Case and any further origin pages.